test/e2e/e2e_suite_test.go

```diff
@@ -79,11 +79,19 @@ var _ = BeforeSuite(func() {
 	}
 
 	nroObj := objects.TestNRO(matchLabels)
+	kcObj := objects.TestKC(matchLabels)
+	pauseMCPs(nroObj.Spec.NodeGroups, true)
```

Comment:
I think we should pause BEFORE creating NRO and KC objects

Reply:
This is not the creation, we're just initializing the structs.
The real creation happens later when an API create call is executed by the controller-runtime client

Reply:
...which we are btw. We do pause before to submit the objects to the cluster.

Reply:
> This is not the creation, we're just initializing the structs here. The real creation happens later when an API create call is executed by the controller-runtime client

aye, I read too hastily. What you're doing here is fine.